src/Microsoft.AspNetCore.Identity/SecurityStampValidator.cs

```diff
@@ -18,13 +19,15 @@ public class SecurityStampValidator<TUser> : ISecurityStampValidator where TUser
     {
         private readonly SignInManager<TUser> _signInManager;
         private readonly IdentityOptions _options;
+        private ISystemClock _clock;
 
         /// <summary>
         /// Creates a new instance of <see cref="SecurityStampValidator{TUser}"/>.
         /// </summary>
         /// <param name="options">Used to access the <see cref="IdentityOptions"/>.</param>
         /// <param name="signInManager">The <see cref="SignInManager{TUser}"/>.</param>
-        public SecurityStampValidator(IOptions<IdentityOptions> options, SignInManager<TUser> signInManager)
+        /// <param name="clock">The system clock.</param>
```

Comment:
Its no longer internal, since you can implement your own ISystemClock and add it to the container (what we do for tests).  But yeah its definied in the Security/Authentication package